### PR TITLE
Allow vTeam creation to everyone

### LINKED ISSUE #613: Allow vTeam creation to everyone

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Feature

## Related issues

Resolve #613

### CODE REVIEW

**Comment by Katrin-kudryash** [comment]
Untitled

File: src/components/CreateGroupModal/CreateGroupModal.tsx [hunk 1]
```diff
@@ -105,19 +107,19 @@ export const CreateGroupModal = ({ visible, onClose }: CreateGroupModalProps) =>
...
-                        value={type}
-                        onChange={(v) => setType(v as GroupType)}
-                    >
-                        {groupTypes.map((t) => (
-                            <FormRadioInput key={t.type} value={t.type} label={t.text} />
-                        ))}
-                    </FormRadio>
-
-                    {nullable(type !== 'virtual', () => (
+                    {data?.access.group.create && (
```
Comment (9teen90nine) [inline src/components/CreateGroupModal/CreateGroupModal.tsx:110]: nullable

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/components/CreateGroupModal/CreateGroupModal.tsx [hunk 2]
```diff
@@ -105,19 +106,19 @@ export const CreateGroupModal = ({ visible, onClose }: CreateGroupModalProps) =>
...
+                            name="type"
+                            value={type}
+                            onChange={(v) => setType(v as GroupType)}
+                        >
+                            {groupTypes.map((t) => (
+                                <FormRadioInput key={t.type} value={t.type} label={t.text} />
+                            ))}
+                        </FormRadio>
+                    ))}
+                    {nullable(type !== 'virtual' && data?.access.group.create, () => (
```
Comment (9teen90nine) [inline src/components/CreateGroupModal/CreateGroupModal.tsx:121]: if you extract `data.access.group.create` into a variable, there will be a copypaste elimination + the code will look much better

**Review by 9teen90nine — APPROVED** [review]

File: src/trpc/router/groupRouter.ts [hunk 3]
```diff
@@ -15,7 +15,11 @@ import { globalAccess } from '../../modules/globalAccess';
 
 export const groupRouter = router({
     create: protectedProcedure.input(createGroupSchema).mutation(({ input, ctx }) => {
-        accessCheck(globalAccess.group.create(ctx.session.user.role));
+        if (input.virtual) {
+            globalAccess.group.createVirtual();
```
Comment (tsumo) [inline src/trpc/router/groupRouter.ts:19]: where is accessCheck? Did you test this with different roles?

**Review by tsumo — CHANGES_REQUESTED** [review]

**Pull request merged**